db/schema.rb

```diff
@@ -42,9 +42,6 @@
     t.index ["blob_id", "variation_digest"], name: "index_active_storage_variant_records_uniqueness", unique: true
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
-  end
-
```

Comment:
This shouldn't be here

Reply:
Still shouldn't be here

Reply:
It seems to be removed

Reply:
No as in this change shouldn't be here - why are you removing the `data_migrations` table?

Reply:
My bad - not sure what happened here.